docs/application/vstools/tools/metadata-element.md

```diff
@@ -0,0 +1,31 @@
+# Metadata element

```

Comment:
First headings should follow Title Case.
```suggestion
# Metadata Element
```

Reply:
thanks! fixed
https://docs1.stg.tizen.org/staging/1681/application/tizen-studio/native-tools/metadata-element/